x-pack/test_serverless/functional/test_suites/search/console_notebooks.ts

```diff
@@ -12,11 +12,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
 
   describe('Console Notebooks', function () {
     before(async () => {
-      await pageObjects.svlCommonPage.loginWithRole('viewer');
-    });
-
-    after(async () => {
-      await pageObjects.svlCommonPage.forceLogout();
+      await pageObjects.svlCommonPage.loginWithPrivilegedRole();
```

Comment:
Can we keep the `view` role login so that we don't give any unnecessary privileges?

Reply:
https://github.com/elastic/kibana/commit/72ae5e891119825a2e077a9b3ea5233a53af400e